src/domains/edit/tests/add-wheelchair-description.e2e-spec.ts

```diff
@@ -0,0 +1,74 @@
+import { type Locator, expect, test } from "@playwright/test";
+import node4544823443Mock from "~/domains/edit/tests/mocks/node-4544823443-osm-mock.json";
+import {
+  getDialog,
+  getEditButton,
+  getMenuItem,
+  mockPlaceInfo,
+} from "~/domains/edit/tests/testUtils";
+import { skipOnboarding } from "../../../../tests/e2e/utils/skipOnboarding";
+
+test.describe("Add wheelchair description in new language", () => {
+  let dialog: Locator;
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto("/");
+    await skipOnboarding(page);
+    await mockPlaceInfo(page);
+    await getEditButton(page, "wheelchair:description").click();
+    await getMenuItem(page, "new-language").click();
+    dialog = await getDialog(page);
+  });
+
+  test("dialog is rendered", async () => {
+    await expect(dialog).toBeVisible();
+  });
+
+  test("dialog has select box when adding a new language", async () => {
+    await expect(dialog.getByRole("combobox")).toBeVisible();
+  });
+
+  test("dialog content is key board navigable", async () => {
+    //TODO
+  });
+
+  test("text area is hidden before language is selected", async ({ page }) => {
+    const textArea = dialog.getByRole("textbox");
+    await expect(textArea).toBeHidden();
+
+    await dialog.getByRole("combobox").click();
+    await page.locator("text=English").click();
```

Comment:
suggestion (non-blocking): I can see the text "English" popping up in different areas of the page in future, especially when we implement a global language picker at some point. This could lead to this test either being flaky or braking. You could make this test more robust by querying children of the dialog instead of locating it on the page level.